core/src/main/resources/schema/app/20180506052935_BUG-102336_Implement_ability_to_pass_custom_parameters.sql

```diff
@@ -0,0 +1,13 @@
+-- // BUG-102336 Implement ability to pass custom parameters
+-- Migration SQL that makes the change goes here.
+
+ALTER TABLE stack ADD COLUMN inputs text;
+ALTER TABLE cluster DROP COLUMN blueprintinputs;
+ALTER TABLE cluster DROP COLUMN blueprintcustomproperties;
```

Comment:
with existence check we could avoid undesired db update when column has already removed for some reason (intentionally with some other migration script, or by mistake), with something like DROP COLUMN IF EXISTS.
same goes for the rollback scripts, some extra check doesn't hurt anybody :)